spread/project.go

```diff
@@ -985,6 +986,19 @@ func (p *Project) Jobs(options *Options) ([]*Job, error) {
 		if !manualTasks && job.Task.Manual {
 			continue
 		}
+
+		// Check the tag in the jon in case a tag is specified in the options
+		if options.Tag != "" {
+			tagMatch := false
+			for tag := range job.Task.Tags {
```

Comment:
when you loop like this you're not getting the tag but the index, so you'd normally have that written as
```go
for i := range job.Task.Tags {
```
or
```go
for _, tag := range job.Task.Tags {
```
In this case I think you want the latter, ...

Reply:
done

Reply:
@chipaca thanks for the review